addons/arsenal/arsenale/fnc_arsenalUSA.sqf

```diff
@@ -920,6 +924,7 @@ _items append (switch (ACE_player getVariable ["TB_rolle", ""]) do
             // Gewehr
             "rhsusf_weap_MP7A2",
             "rhsusf_weap_MP7A2_grip3",
+	    "TB_weap_Vector",
```

Comment:
bekomme das hier direkt in Github nicht eingerückt und der "Edit Version" sieht es richtig aus

Reply:
das gleiche auch in Zeile 751, 753 und 756

Reply:
@Braincrushy keine Tabs benutzen!

Reply:
check war mir nicht bekannt keine Tabs in Zukunft
irgendeine Idee weshalb der validate SQF noch nen Error gibt
Exit code 10 alleine sagt mir leider nicht viel.

Reply:
@Braincrushy hab die Tabs jetzt rausbekommen musst in deinem Editor der Wahl einstellen das wenn du Tab drückst das er 4 Leerzeichen macht.